CHANGELOG.md

```diff
@@ -1,3 +1,59 @@
+- **Breaking change**: Updated controller-runtime to v0.6.0. ([#2918](https://github.com/operator-framework/operator-sdk/pull/2918))
+- Updated controller-tools to v0.3.0. ([#2918](https://github.com/operator-framework/operator-sdk/pull/2918))
+- Updated helm to v3.2.0. ([#2918](https://github.com/operator-framework/operator-sdk/pull/2918))
+
+### Removals
+
+- **Breaking change**: The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated and it will no longer scaffold the `/bin/ao-logs` which was using it to print the Ansible logs in the side-car since the side-car ansible container was removed in the previous versions. ([#2852](https://github.com/operator-framework/operator-sdk/pull/2852))
+- **Breaking change**: Removed automatic migration of helm releases from v2 to v3. ([#2918](https://github.com/operator-framework/operator-sdk/pull/2918))
+- **Breaking change**: Removed support for deprecated helm release naming scheme. ([#2918](https://github.com/operator-framework/operator-sdk/pull/2918))
+
+### Deprecations
+
+- Deprecate 'run --olm' mode. Use 'run packagemanifests' instead. ([#3016](https://github.com/operator-framework/operator-sdk/pull/3016))
+- Deprecate '--kubeconfig' flag on the 'cleanup' subcommand. Use 'run packagemanifests' instead. ([#3067](https://github.com/operator-framework/operator-sdk/pull/3067))
+- Deprecate 'run --local' mode. Use 'run local' instead. ([#3067](https://github.com/operator-framework/operator-sdk/pull/3067))
+
+### Bug Fixes
+
+- Fixes issue where the `helm.operator-sdk/upgrade-force` annotation value for Helm based-operators is not parsed. ([#2894](https://github.com/operator-framework/operator-sdk/pull/2894))
+- In 'run --olm', package manifests format must be replicated in a pod's file system for consistent registry initialization. ([#2964](https://github.com/operator-framework/operator-sdk/pull/2964))
+- the internal OLM client retrieves existing OLM versions correctly now that the returned list of CSVs is indexed properly. ([#2969](https://github.com/operator-framework/operator-sdk/pull/2969))
+- The Ansible Operator proxy will now return a 500 if it cannot determine whether a resource is virtual or not, instead of continuing on and skipping the cache. This will prevent resources that should have ownerReferences injected from being created without them, which would leave the user in a state that cannot be recovered without manual intervention. ([#3112](https://github.com/operator-framework/operator-sdk/pull/3112))
+- The Ansible Operator proxy no longer will attempt to cache non-status  subresource requests. This will fix the issue where attempting to get Pod logs returns the API Pod resource instead of the log contents. ([#3103](https://github.com/operator-framework/operator-sdk/pull/3103))
+- do not generate a package manifest when 'generate csv --make-manifests=true'. ([#3014](https://github.com/operator-framework/operator-sdk/pull/3014))
```

Comment:
Nit:
```suggestion
- Do not generate a package manifest when 'generate csv --make-manifests=true'. ([#3014](https://github.com/operator-framework/operator-sdk/pull/3014))
```

Reply:
The changelog generator should impose some formatting rules like this.

Reply:
Yeah, or better yet we try to catch this when the Fragment PR is added.
So that the release manager doesn't have to fix it all up.